OIPA/iati_organisation/parser/organisation_2_03.py

```diff
@@ -706,6 +708,164 @@ def iati_organisations__iati_organisation__recipient_org_budget__budget_line__na
         self.add_narrative(element, recipient_org_budget_line)
         return element
 
+    def iati_organisations__iati_organisation__recipient_region_budget(
+            self, element):
+        status = element.attrib.get("status")
+        if status:
+            status = self.get_or_none(codelist_models.BudgetStatus,
+                                      code=status)
+            if status is None:
+                raise FieldValidationError(
+                    "RecipientRegionBudget",
+                    "status",
+                    "not found on the accompanying codelist.",
+                    None,
+                    None,
+                )
+        else:
+            status = codelist_models.BudgetStatus.objects.get(code=1)
+        recipient_region = element.xpath("recipient-region")
+        if len(recipient_region) is not 1:
+            raise ParserError("RecipientRegionBudget",
+                              "recipient-region",
+                              "must occur once and only once.")
+        narrative = recipient_region[0].xpath("narrative")
```

Comment:
Correct me if I'm wrong, I don't understand why are you checking the existence of child methods not in specific parser methods for these particular elements but there. Also, according to [this](http://reference.iatistandard.org/203/organisation-standard/iati-organisations/iati-organisation/recipient-region-budget/) and [this](http://reference.iatistandard.org/203/organisation-standard/iati-organisations/iati-organisation/recipient-region-budget/recipient-region/), those elements should not contain a `<narrative>` element.